odoo/addons/base/views/res_users_views.xml

```diff
@@ -212,7 +212,7 @@
                             <field name="partner_id" required="0" readonly="1"/>
                           </div>
                         </div>
-                        <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "image_128"}'/>
+                        <field name="image_1920" widget='avatar' class="oe_avatar" options='{"preview_image": "image_128"}'/>
```

Comment:
What happens on the partner view of the user? Do we see another default image because the avatar route only works for users? That would be a bit confusing to me.
What about the employee view?